src/components/TaskCard/AddTaskCard.jsx

```diff
@@ -0,0 +1,69 @@
+import React, { useState } from "react";
+import { useDispatch } from "react-redux";
+import { v1 as uuidv1 } from "uuid";
+import { sanitizeText } from "utils/helpers/sanitizeText.js";
+import PropTypes from "prop-types";
```

Comment:
maintain import order, library imports should be moved to the top.

```suggestion
import PropTypes from "prop-types";
import { sanitizeText } from "utils/helpers/sanitizeText.js";
```